policyengine_us/parameters/gov/states/md/tax/income/deductions/standard/max.yaml

```diff
@@ -18,15 +20,20 @@ metadata:
 JOINT:
   2018-01-01: 4_500
   2021-01-01: 4_700
+  2023-01-01: 5_150
 HEAD_OF_HOUSEHOLD:
   2018-01-01: 4_500
   2021-01-01: 4_700
```

Comment:
check 2022 file for the 2022 vlaues:
https://marylandtaxes.gov/forms/22_forms/Resident_Booklet.pdf

Also add uprating metadata:
example:
  uprating:
    parameter: gov.irs.uprating
    rounding:
      type: downwards
      interval: 50

Reply:
same for the min file